test/export/test_fullgraph_package.py

```diff
@@ -0,0 +1,49 @@
+# Owner(s): ["oncall: export"]
+
+import os
+import pathlib
+import unittest
+
+import torch
+from torch.testing._internal.common_utils import run_tests, TEST_CUDA, TestCase
+
+
+class TestFullgraphPackage(TestCase):
+    def setUp(self):
+        if not os.path.exists(os.path.expandvars("/tmp/torchinductor_$USER/")):
+            os.makedirs(os.path.expandvars("/tmp/torchinductor_$USER/"))
+
+    def tearDown(self):
+        super().tearDown()
+        pathlib.Path(self.path()).unlink(missing_ok=True)
+
+    def path(self):
+        return os.path.expandvars(f"/tmp/torchinductor_$USER/model_{self.id()}.pt2")
+
+    @unittest.skipIf(not TEST_CUDA, "requires cuda")
+    def test_fullgraph_package_basic_function(self):
+        def f(a, b):
+            for i in range(1000):
+                a = a + b * i
+            return a
+
+        inputs = torch.randn(3, device="cuda"), torch.randn(3, device="cuda")
+        expected = f(*inputs)
+        f = torch.compile(f, fullgraph=True, name=self.id())
+        p, *flags = torch.compiler._enable_fullgraph_package(path=self.path())
```

Comment:
This API seems a bit clunky, could it be done with a non-mode like API?  I don't understand what the varargs `*flags` part is.

Maybe something like:
```py
f = torch.compiler.cached_compile(f)
...
f.save_compile_artifacts(path)


f.load_compile_artifacts(path)
```

Or even just add `save_compile_artifacts` and `load_compile_artifacts` methids to return value of `torch.compile`

Reply:
Happy to chat in more detail if you need more detail on how to implement that.